app/src/main/java/com/example/ridemetricsdksample/MainActivity.java

```diff
@@ -1,29 +1,96 @@
 package com.example.ridemetricsdksample;
 
 import android.os.Bundle;
+import android.os.Environment;
 import android.support.v7.app.ActionBarActivity;
-import android.view.Menu;
-import android.view.MenuItem;
+import android.util.Log;
 import android.widget.Toast;
 
+import com.ridemetric.sdk.DriverRegistrationInfo;
 import com.ridemetric.sdk.RideMetric;
 import com.ridemetric.sdk.RidemetricError;
 import com.ridemetric.view.R;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Locale;
+
+//import com.ridemetric.sdk.RidemetricError;
```

Comment:
It's better to remove commented and dead code before present the sample to clients.